auth/providers/azure/graph/graph_test.go

```diff
@@ -107,11 +107,12 @@ func TestLogin(t *testing.T) {
 		validBody := `{
   "token_type": "Bearer",
   "expires_in": 3599,
-  "access_token": "%s"
+  "access_token": "%s",
+  "expires_on": %d
 }`
-		ts, u := getAuthServerAndUserInfo(http.StatusOK, fmt.Sprintf(validBody, validToken), "jason", "bourne")
+		expiresOn := time.Now().Add(time.Second * 3599).Unix()
+		ts, u := getAuthServerAndUserInfo(http.StatusOK, fmt.Sprintf(validBody, validToken, expiresOn), "jason", "bourne")
```

Comment:
Can you add a check to make sure the expiration time is exactly the `expiresOn` value minus 5min?
Currently the test doesn't validate the change in behavior.

Reply:
Updated to test the expected behavior